src/sagemaker/jumpstart/curated_hub/utils.py

```diff
@@ -164,3 +185,114 @@ def create_hub_bucket_if_it_does_not_exist(
     )
 
     return bucket_name
+
+def tag_hub_content(hub_content_arn: str, tags: List[TagsDict], session: Session) -> None:
+    session.add_tags(
+      ResourceArn=hub_content_arn,
+      Tags=tags
+    )
+    JUMPSTART_LOGGER.info(f"Added tags to HubContentArn %s: %s", hub_content_arn, TagsDict)
```

Comment:
this may emit too many logs, especially for hubs with many models. consider removing, or logging once after all the tagging is completed

Reply:
This only logs when a model in the CuratedHub has been identified as vulnerable or deprecated. I think it's important to log for each one

Reply:
i thought this logs for each hub content